### DISCUSSION TITLE
A bypass to use (Official)

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#131] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
You can use `javascript:(function(){window.onbeforeunload=function(){return '';}})();` to access blocked sites. How you do it: press search+esc, crash goguardian, get ready to quickly visit the site. Keep goguardian crashed, visit the site, spam the bookmarklet then just leave it until it asks if you want to leave. press cancel. this will happen every so often, press cancel every time. some sites won't let you redirect to anythiing, but if you're on youtube or something, don't worry just press cancel when you visit a video or look at it in miniplayer

**Comment by synapt0** [comment]
I have securly so it doesn't work for me.

**Reply by ghost** [reply]
same thing, isn't it?

**Reply by Alex2Mayhem** [reply]
> same thing, isn't it?

What is the "site"

**Reply by ghost** [reply]
any site

**Comment by Sxri** [comment]
Doesn't work for other extensions, ex: Securely, Border-less Classroom, etc

**Reply by ghost** [reply]
why not? what does your extension do when you visit a blocked site?

**Reply by Sxri** [reply]
It gets blocked by the wifi if the border-less classroom extention doesnt work. VPNs dont work cause we cant add extensions or apps, and currently, my phone is broken so i cant use hotspot. plus my hotspot is very laggy, so is everyone elses in the school.

**Reply by Aka-but-nice** [reply]
> why not? what does your extension do when you visit a blocked site?
Securly is based on DNS it always runs, unlike GoGuardian which has a 2-second wait time until it's blocked, for securly it's different for example when you go to 1v1.lol it immediately goes to its blocked page we would literally have to be a AI to do it.

**Comment by Alex2Mayhem** [comment]
> {

Simple instructions?

**Reply by ghost** [reply]
what? just put the bookmarklet in and use it

**Reply by Alex2Mayhem** [reply]
> what? just put the bookmarklet in and use it

 Yeah but it seems to not be doing anything. so what am i missing? i've even ran it in a code software.
 
 Search + Esc opens task manager. But how do you crash it?

**Reply by ghost** [reply]
go to task manager, find goguardian at the bottom, press "end task", keep doing it whilst visiting the site, then spam the bookmarklet. It will ask if you want to close it, press no. I'll attach a video hold on

**Reply by ghost** [reply]
https://user-images.githubusercontent.com/77510164/197798677-27857648-6643-423f-a89e-216fc64c5476.mp4

**Reply by Alex2Mayhem** [reply]
> go to task manager, find goguardian at the bottom, press "end task", keep doing it whilst visiting the site, then spam the bookmarklet. It will ask if you want to close it, press no. I'll attach a video hold on

i'm pretty sure. GG. pretty much stops you from hitting end task on things on task manager. But i'd still like to see that video. 
![image](https://user-images.githubusercontent.com/104642381/197798816-a26144da-dd74-4826-ac04-587043cbd879.png)
That image shows you can't end.

**Reply by ghost** [reply]
I posted the vid

**Reply by Alex2Mayhem** [reply]
> I posted the vid

Okay yeah, so you can't end processes with task manager.

**Reply by Alex2Mayhem** [reply]
i found a source view to view the code of chrome webstorex. i found a few links in its code. they could be alternates i just have to go through them. 
![image](https://user-images.githubusercontent.com/104642381/197801380-04e6d77a-06fb-4dc8-95c5-6ecef2b639a7.png)

**Reply by ghost** [reply]
you do that

**Reply by Alex2Mayhem** [reply]
> you do that

Okay, ill tell you if i get a hint on how to fix it or a backdoor.

**Reply by ghost** [reply]
thnx

**Reply by Alex2Mayhem** [reply]
Use google's dev own code against them possibly? https://developer.chrome.com/docs/extensions/reference/management/#method-uninstall
 https://developer.chrome.com/docs/extensions/reference/management/
![image](https://user-images.githubusercontent.com/104642381/197809013-17f03254-1621-43f5-984b-c93cc1e4161f.png)

**Reply by ghost** [reply]
u do that

**Reply by Meta0Music** [reply]
> Use google's dev own code against them possibly? https://developer.chrome.com/docs/extensions/reference/management/#method-uninstall https://developer.chrome.com/docs/extensions/reference/management/ ![image](https://user-images.githubusercontent.com/104642381/197809013-17f03254-1621-43f5-984b-c93cc1e4161f.png)

![image](https://user-images.githubusercontent.com/114110114/197830474-f623cb0e-75c7-4f0c-92ed-5f2ab04ddd32.png)
It says it will not work if you already can't uninstall the extension. so idk what you on about

**Reply by ghost** [reply]
guys idrc 😅

**Reply by Alex2Mayhem** [reply]
> > Use google's dev own code against them possibly? https://developer.chrome.com/docs/extensions/reference/management/#method-uninstall https://developer.chrome.com/docs/extensions/reference/management/ ![image](https://user-images.githubusercontent.com/104642381/197809013-17f03254-1621-43f5-984b-c93cc1e4161f.png)
> 
> ![image](https://user-images.githubusercontent.com/114110114/197830474-f623cb0e-75c7-4f0c-92ed-5f2ab04ddd32.png) It says it will not work if you already can't uninstall the extension. so idk what you on about

 Thats what you think

**Reply by ghost** [reply]
oh my lord

**Comment by ghost** [comment]
UPDATE: see [this repo](https://github.com/yeeteeyt/GoGuardian-bypass) for the bypass
